NAS-112059 / 21.10 / Use iX Blue + Dark Blue for official Apps

Ornias1993 opened the pull request:
As explained in NAS-112059 people are too easily confused between official and community Apps at the moment.

This PR gives two color options:
- Official in a combination of iX Blue and Dark blue
- Community in Yellow and the original disgusting dark yellow.

For Bluefin this, most likely, would need to be redesigned to deal with more than just two color options, but for Anglefish this solves the problem without too much code redundency imho.

**Before:**
![image](https://user-images.githubusercontent.com/7613738/133502749-13732fea-820c-465c-b72e-ac5ab1a5c6c8.png)

**After:**
![image](https://user-images.githubusercontent.com/7613738/134148645-23879636-c93b-4c56-afe4-347d24ae8924.png)

bugclerk commented:
Jira URL: https://jira.ixsystems.com/browse/NAS-112059

RehanY147 reviewed (changes requested):
Maybe we should try white text on top the blue background?

Ornias1993 commented:
> Maybe we should try white text on top the blue background?

That would become extremely flashy, which means making the blue darker, which in turn makes it stand out extremely from the dark yellow :(

i'll try tweaking the colors some more to look if I can somehow compensate the black-on-blue a bit :)

Ornias1993 commented:
> Maybe we should try white text on top the blue background?

I've changed the PR a bit:
- Color of the train-label is now standardised for both the blue and yellow labels (it uses the same black overlay color)
- Reduced the amount of duplicate CSS code
- Also implemented the new scheme in the "Installed Apps" section

RehanY147 commented:
@Ornias1993 Please provide the latest screenshots with this

RehanY147 reviewed (commented):
.

Ornias1993 commented:
> @Ornias1993 Please provide the latest screenshots with this

Understandable, with my latest changes i've edited the PR description and screenshots right away accordingly :)

RehanY147 commented on src/app/pages/applications/catalog/catalog.component.html:
```diff
@@ -16,8 +16,16 @@
           </div>
-          <div class="badge-area">
-            <div class="partner-badge capitalized-text">
+          <div class="badge-area" style="background-color: var(--primary)" *ngIf="item.catalog.label == 'OFFICIAL'">
```
We should put the styles in CSS classes

Ornias1993 replied:
I've moved it to an additional class that only overrides the color.

RehanY147 reviewed (approved):
Looks good. Thank you!

RehanY147 reviewed (approved):
Thank you!
